pkg/resources/istiocoredns/kubedns.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 Banzai Cloud.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package istiocoredns
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-logr/logr"
+	"github.com/goph/emperror"
+	apiv1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/banzaicloud/istio-operator/pkg/k8sutil"
+)
+
+// Add/Remove .global to/from kube-dns configmap
+func (r *Reconciler) reconcileKubeDNSConfigMap(log logr.Logger, desiredState k8sutil.DesiredState) error {
```

Comment:
In the [docs](https://istio.io/docs/setup/kubernetes/install/multicluster/gateways/#setup-dns) there are separate examples for editing the ConfigMaps in case of `kube-dns` and `CoreDNS`. In this solution, you support `kube-dns`, but from Kubernetes 1.11 `CoreDNS` seems to be the [default](https://kubernetes.io/docs/tasks/administer-cluster/coredns/#installing-kube-dns-instead-of-coredns-with-kubeadm).

Could we support both? Or is there a reason why we should only take care of `kube-dns`?